channels/applicationinsights-channel-js/src/EnvelopeCreator.ts

```diff
@@ -162,7 +170,7 @@ export abstract class EnvelopeCreator {
 
     protected _logger: IDiagnosticLogger;
 
-    abstract Create(logger: IDiagnosticLogger, telemetryItem: ITelemetryItem): IEnvelope;
+    abstract Create(logger: IDiagnosticLogger, telemetryItem: ITelemetryItem, customDefinedValue?: any): IEnvelope;
```

Comment:
nit: ```customUndefinedValue```

Reply:
I mean the value is a custom defined value to replace the "undefined", should this be "customUndefinedValue"?

Reply:
I think it should be ```customUndefinedValue``` as this is the value that will be used when the value is ```undefined```/